# HG changeset patch
# User Rik <rik@octave.org>
# Date 1344629351 25200
#      Fri Aug 10 13:09:11 2012 -0700
# Node ID 142e377e7e28911cd2e2650b1a67fcd243981b49
# Parent  9cc337ced51a68ebc86fcb3559aedf1426bd69e2
configure.ac: Don't run OCTAVE_CARBON_CGDISPLAYBITSPERPIXEL unless Carbon Framework is present.

* configure.ac: Don't run OCTAVE_CARBON_CGDISPLAYBITSPERPIXEL unless
Carbon Framework is present.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -1990,17 +1990,19 @@ esac
 AC_CHECK_FUNCS(getrusage)
 case "$canonical_host_type" in
   *-*-cygwin*)
     AC_DEFINE(RUSAGE_TIMES_ONLY, 1, [Define to 1 if your struct rusage only has time information.])
   ;;
 esac
 
 ## Check for CGDisplayBitsPerPixel function on Mac OSX systems with Carbon
-OCTAVE_CARBON_CGDISPLAYBITSPERPIXEL
+if test $have_framework_carbon = "yes"; then
+  OCTAVE_CARBON_CGDISPLAYBITSPERPIXEL
+fi
 
 OCTAVE_SMART_PUTENV
 
 AC_CHECK_FUNCS(getpwnam, [], [AC_CHECK_LIB(sun, getpwnam)])
 
 AC_FUNC_CLOSEDIR_VOID
 
 ## Check return type of matherr() 
